lib/hexo/index.js

```diff
@@ -212,6 +212,16 @@ class Hexo extends EventEmitter {
   }
 
   init() {
+    // Load only the relevant plugins in hexo clean
+    if (this.env.cmd.startsWith('c')) {
+      this.env.init = true;
+      return Promise.all([
+        this.extend.console.register('clean', 'Remove generated files and cache.', require('../plugins/console/clean')),
+        require('../plugins/filter')(this),
+        require('./load_plugins')(this)
+      ]);
```

Comment:
I would suggest always registers all internal `console` plugins.

```suggestion
      return Promise.all([
       require('../plugins/console')(this),
        require('../plugins/filter')(this),
        require('./load_plugins')(this)
      ]);
```

Reply:
All other internal `console` plugins are not relevant to `clean`.
Or is it for cleaner code?

Reply:
@curbengh That makes sense.